### PR TITLE
[WIP] Datastore example

### PR DESCRIPTION
**Pull request by ian-r-rose**
Very rough start on a node server + server adapter for the datastore example. With @jasongrout

### CODE REVIEW

File: examples/example-datastore/dsServer.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
+
+const WebSocket = require('ws');
+const patchDB = new Map();
+numStores = 0;
```
Comment (jasongrout) [inline examples/example-datastore/dsServer.js]: Missing `let`

**Comment by jasongrout** [comment]
Looks like there are some packages missing from the package.json too.

**Comment by sccolbert** [comment]
@ian-r-rose is this obsolete now?

**Comment by ian-r-rose** [comment]
Yes, I may reuse some of it later, but it can be closed.

**Pull request closed without merging**